CommonCompilerConfig.cmake

```diff
@@ -110,14 +111,24 @@ set(EXTERNAL_LINKER_FLAGS_OPTIMIZED)
 set(EXTERNAL_STATICLIB_CMAKE_DEFINITIONS_OPTIMIZED)
 set(EXTERNAL_STATICLIB_CONFIGURE_DEFINITIONS_OPTIMIZED)
 
+
 # Identify platform "bitness".
-if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-  set(BITNESS 64)
+if(NOT BITNESS)
+  if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+    set(BITNESS 64)
+  else()
+    set(BITNESS 32)
+  endif()
+endif()
+if(NOT INTERNAL_CPP_STANDARD)
```

Comment:
keep this on CMake, there's some CMake_CPP_version c++11)

Reply:
I actually let this be passed in from the SuperBuild.  The user can set this on the command line as an override.  Eventually  I will set this to C++17 as the default. but I have to get rid of some stuff first.